src/ralph/backends/data/swift.py

```diff
@@ -0,0 +1,380 @@
+            dict: The next object details. (If `details` is True.)
+
+        Raises:
+            BackendException: If a failure occurs.
+        """
+        if target is None:
+            target = self.default_container
+
+        archives_to_skip = set()
+        if new:
+            archives_to_skip = set(self.get_command_history(self.name, "read"))
+
+        try:
+            _, objects = self.connection.get_container(
+                container=target, full_listing=True
+            )
+        except ClientException as err:
+            msg = "Failed to list container %s: %s"
+            logger.error(msg, target, err.msg)
+            raise BackendException(msg % (target, err.msg)) from err
+
+        for obj in objects:
+            if new and obj in archives_to_skip:
+                continue
+            yield self._details(target, obj) if details else obj
+
+    @enforce_query_checks
+    def read(
+        self,
+        *,
+        query: Union[str, BaseQuery] = None,
+        target: str = None,
+        chunk_size: Union[None, int] = 500,
+        raw_output: bool = False,
+        ignore_errors: bool = False,
+    ) -> Iterator[Union[bytes, dict]]:
+        """Read objects matching the `query` in the `target` container and yields them.
+
+        Args:
+            query: (str or BaseQuery): The query to select objects to read.
+            target (str or None): The target container name.
+                If `target` is `None`, a default value is used instead.
+            chunk_size (int or None): The number of records or bytes to read in one
+                batch, depending on whether the records are dictionaries or bytes.
+            raw_output (bool): Controls whether to yield bytes or dictionaries.
+                If the objects are dictionaries and `raw_output` is set to `True`, they
+                are encoded as JSON.
+                If the objects are bytes and `raw_output` is set to `False`, they are
+                decoded as JSON by line.
+            ignore_errors (bool): If `True`, errors during the read operation
+                are be ignored and logged. If `False` (default), a `BackendException`
+                is raised if an error occurs.
+
+        Yields:
+            dict: If `raw_output` is False.
+            bytes: If `raw_output` is True.
+
+        Raises:
+            BackendException: If a failure during the read operation occurs and
+                `ignore_errors` is set to `False`.
+            BackendParameterException: If a backend argument value is not valid.
+        """
+        if query.query_string is None:
+            msg = "Invalid query. The query should be a valid archive name."
+            logger.error(msg)
+            if not ignore_errors:
+                raise BackendParameterException(msg)
+
+        target = target if target else self.default_container
+
+        logger.info("Getting object: %s from container: %s", query.query_string, target)
```

Comment:
The string is not very readable, as the `suqery_string` does not list names of objects, right ? 
I would propose something simple: 
```suggestion
        logger.info("Getting object from container: %s (query string: %s)", target, query.query_string)
```

Reply:
Indeed, thanks!